library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -100,7 +100,7 @@
     public static OktaAppAuth getInstance(@NonNull Context context) {
```

Comment:
As long as context is a WeakReference now I think we can add context reinitialization if it is null. Something like:
 OktaAppAuth oktaAppAuth = INSTANCE_REF.get().get();
```
        if (oktaAppAuth == null) {
            oktaAppAuth = new OktaAppAuth(context);
            INSTANCE_REF.set(new WeakReference<>(oktaAppAuth));
        } else if (oktaAppAuth.mContext.get() == null) {
            oktaAppAuth.mContext = new WeakReference<>(context);
        }

        return oktaAppAuth;

```